packages/attribute-registry-readmodel-writer/src/index.ts

```diff
@@ -47,16 +47,9 @@ await consumer.subscribe({
 });
 
 async function processMessage(message: KafkaMessage): Promise<void> {
-  try {
-    await handleMessage(
-      decodeKafkaMessage(message, AttributeEvent),
-      attributes
-    );
+  await handleMessage(decodeKafkaMessage(message, AttributeEvent), attributes);
 
-    logger.info("Read model was updated");
-  } catch (e) {
-    logger.error(`Error during message handling ${e}`);
-  }
+  logger.info("Read model was updated");
```

Comment:
I know it is out of scope, but may I ask you to add details here like in other consumers? 🙏 
```suggestion
  logger.info("Read model was updated. Partition number: ${partition}. Offset: ${message.offset}");
```

Reply:
Updated